.circleci/config.yml

```diff
@@ -1,5 +1,9 @@
 # anchor refs to be used elsewhere
 refs:
+  constants:
+  # TODO upload to hasura docker hub:
+  # - &server_builder_image hasura/graphql-engine-server-builder:2020-01-14
+  - &server_builder_image jberryman/graphql-engine-server-builder-8.10:2020-04-29
```

Comment:
Would it be possible to upload to docker first? Or is there a reason for doing it this way?

Reply:
@0x777 I don't have credentials (I don't think) to push to hasura's docker hub, and forgot to ping someone about this TODO. Do you want to do this before merge? What's the best way to get this into hasura namespace?

Reply:
@jberryman I have added your the hasura org on docker hub. I have also re-tagged and pushed the image to hasura.